packages/material-ui-lab/src/Rating/Rating.test.js

```diff
@@ -66,4 +67,25 @@ describe('<Rating />', () => {
     });
     expect(container.querySelectorAll(`.${classes.iconHover}`).length).to.equal(2);
   });
+  describe('props: IconContainerComponent', () => {
+    it('should render with custom icons', () => {
+      function IconContainer() {
+        return (
+          <span>
+            <Star />
+          </span>
+        );
+      }
+      const { container } = render(
+        <Rating
+          name="read-only"
+          value={1}
+          max={5}
+          IconContainerComponent={IconContainer}
+          readOnly
+        />,
+      );
+      expect(container.querySelectorAll(`.${classes.icon}`).length).to.equal(5);
```

Comment:
With the spread, I don't think that it should be working, does the test pass?

Reply:
updated this test.